policyengine_us/parameters/gov/states/la/tax/income/exemptions/amount/blind_or_disabled.yaml

```diff
@@ -0,0 +1,15 @@
+description: Louisiana provides an exemption of this amount for disabled and blind filers.
+
+values:
+  2014-01-01: 1_000
+metadata:
+  reference:
+    - title: Louisiana Laws Revised Statutes, Title 47 — Revenue and taxation, RS 47:79 — Credits of individuals against net income §79, A, (2)
+      href: https://www.legis.la.gov/legis/Law.aspx?d=102518
+    - title: "Form IT-540-WEB "
+      href: https://revenue.louisiana.gov/TaxForms/IT540WEB(2022)%20F%20D2.pdf
+    - title: "General Information for Filing Your 2022 Louisiana Resident Individual Income Tax Return"
+      href: https://revenue.louisiana.gov/TaxForms/IT540iWEB(2022)D1.pdf
```

Comment:
Same thing - I don't see $1,000 in this document. Always cite page numbers to facilitate verification of parameter values.